api/namex/services/name_request/auto_analyse/name_analysis_director.py

```diff
@@ -285,24 +289,25 @@ def execute_analysis(self):
                 analysis.append(check_words_to_avoid)
                 return analysis
 
-            self.set_skip_search_conflicts(True)
-            check_name_is_well_formed = builder.check_name_is_well_formed(
-                self._dict_name_words,
-                self._list_dist_words,
-                self._list_desc_words,
-                self.compound_descriptive_name_tokens,
-                self.processed_name,
-                self.name_original_tokens
-            )
-            if check_name_is_well_formed.result_code in (
-                    AnalysisIssueCodes.ADD_DISTINCTIVE_WORD, AnalysisIssueCodes.ADD_DESCRIPTIVE_WORD) and \
-                    self.entity_type not in (BCUnprotectedNameEntityTypes.list()) and \
-                    not check_name_is_well_formed.is_valid:
-                analysis.append(check_name_is_well_formed)
-            elif check_name_is_well_formed.result_code == AnalysisIssueCodes.CORPORATE_CONFLICT:
+            if auto_analyze_config in ('WELL_FORMED_NAME', 'EXACT_MATCH', 'SEARCH_CONFLICTS'):
```

Comment:
Loosk like it does not include check_words_to-avoid.  I think the if statement shoudl inlcud ethat as well?  SO just needs t be moved up.  What do you think?